src/content/protocols/curve.md

```diff
@@ -0,0 +1,529 @@
+---
+protocol: "Curve Finance"
+website: "https://curve.fi/"
+x: "https://x.com/CurveFinance"
+github: ["https://github.com/CurveFi"]
+defillama_slug: ["curve-finance"]
+chain: "Ethereum"
+stage: 1
+reasons: []
+risks: ["L", "H", "L", "M", "L"]
+author: ["Saint Rat"]
+submission_date: "2025-04-26"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Curve Finance is a multi-faceted DeFi protocol encompassing a decentralized exchange (DEX) primarily focused on stablecoin and correlated-asset swaps, lending markets (Llamalend), and a decentralized stablecoin (crvUSD). Its governance is managed via the Curve DAO using a vote-escrowed token model (veCRV).
+
+# Overview
+
+## Chain
+
+Curve Finance's core governance (DAO) resides and operates on the Ethereum mainnet. While Curve protocol components are deployed across numerous EVM-compatible chains, this review focuses exclusively on the Ethereum mainnet deployment. Cross-chain messaging contracts and deployments on other chains are outside the scope of this analysis. As Ethereum is a mature Layer 1 blockchain, it meets the criteria for the lowest risk category.
+
+> Chain score: Low
+
+## Upgradeability
+
+The Curve protocol incorporates significant upgradeability mechanisms, primarily managed by the Curve DAO via the `OwnershipAgentProxy`. This agent possesses administrative privileges over key contracts, including deployment factories, core DAO components, revenue distribution, and parameter setting contracts.
+
+Key aspects include:
+
+1.  **Contract Logic Upgrades:** The DAO can modify implementation contracts for core functions and future deployments.
```

Comment:
Could you rephrase some of the elements without using bullet points? A simple paragraph that mentions for each element => what can be done (eg. contracts can be upgraded), and does it lead to loss of funds / loss of unclaimed yield.

There is no need to dive into technical details in this section; it aims to provide an easy-to-understand overview of each criteria.